external/avro/src/main/scala/org/apache/spark/sql/avro/package.scala

```diff
@@ -36,4 +40,27 @@ package object avro {
     @scala.annotation.varargs
     def avro(sources: String*): DataFrame = reader.format("avro").load(sources: _*)
   }
+
```

Comment:
Why these two functions are not a part of ```org.apache.spark.sql.functions```?

Reply:
because avro data source is an external package like kafka data source. It's not available in `org.apache.spark.sql.functions`

Reply:
Thanks guys for your explanation!